CLAUDE.md

```
@@ -0,0 +1,105 @@
+# Brain
+
+Brain (`r2luna/brain`) is a Laravel package that organizes business logic into three core concepts: **Processes**, **Tasks**, and **Queries**.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)